ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -30,7 +30,7 @@ export BITSTREAM="--offline --list ${SHA}"
 
 # We will lose serial access when we reboot, but if tests fail we should reboot
 # in case we've crashed the UART handler on the CW310's SAM3U
-trap 'python3 ./util/fpga/cw310_reboot.py' EXIT
+trap './bazelisk.sh run //sw/host/opentitantool fpga reset' EXIT
```

Comment:
Why not `ci/bazelisk.sh`?

Reply:
I was copying line 37 from this fine. Is `ci/bazelisk.sh` more correct?

Reply:
I think so.

ci/bazelisk.sh has some things that make its output better suited to logging and tends to be cached and used within CI.

The output on line 37 is being piped to xargs so it benefits from having more standard output.

Reply:
Thanks for the explanation! Fixed.